tests/components/laundrify/test_coordinator.py

```diff
@@ -0,0 +1,50 @@
+"""Test the laundrify coordinator."""
```

Comment:
We shouldn't test the coordinator directly. Test the binary sensor platform instead by setting up the integration and asserting the state of the entity in the core state machine.